### ISSUE TITLE
Test for WCAG success criterion 1.3.2: Meaningful Sequence

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#168]
When the sequence in which content is presented affects its meaning, a correct reading sequence can be programmatically determined.
## Scope selector

HTML page
## Applicability

When the sequence in which content is presented affects its meaning
## HTML Failures
- [x] F49: Using an HTML layout table that does not make sense when linearized (#169)
- [x] F32: Using white space characters to control spacing within a word (whiteSpaceInWord)
- [x] F1: Changing the meaning of content by positioning information with CSS  (#170)
### Plain text Failures

Question: is this in scope for HTML?
- [x] F34: Using white space characters to format tables in plain text content (tabularDataIsInTable)
- [x] F33: Using white space characters to create multiple columns in plain text content (tabularDataIsInTable)
## Techniques
- [x] G57: Ordering the content in a meaningful sequence (scope: for all the content in the Web page #91 )

OR
- [x] G57: Ordering the content in a meaningful sequence (scope: for sequences #91)
  AND Marking sequences in the content as meaningful with
- [x] H34: Using a Unicode right-to-left mark (RLM) or left-to-right mark (LRM) to mix text direction inline (languageUnicodeDirection)
- [x] H56: Using the dir attribute on an inline element to resolve problems with nested directional runs (#72)
- [ ] C6: Positioning content based on structural markup (CSS) (#74)
- [x] C8: Using CSS letter-spacing to control spacing within a word (#73)

OR
- [x] C27: Making the DOM order match the visual order (CSS) (#171)
## Relationship
- C8 is counterpart to F32
- C8 and F32 can also be not applicable if there is no spacing. So when C8 pass, F32 pass as well, when C8 is n/a, F32 is n/a as well, and when C8 fails, F32 fails as well.
- C6 is counterpart to F1
- Can C6/F1 be not applicable?
- C8, H34 and H56 can all be not applicable. Is then the success criterion met only when C6 is used? And are the techniques OR or AND?
- G57 has to granularity levels: sequences and page. 
- F49 has a applicablity criterion: layout table
- G57 on page level implies using layout tables, with F49 as counterpart: "At a larger granularity, controlling the placement of blocks of content in an HTML document using layout tables may produce a rendering in which related information is positioned together visually, but separated in the content stream. Since layout tables are read row by row, if the caption of an illustration is placed in the row following the illustration, it may be impossible to associate the caption with the image."
- the difference of the two options are not clear to me. Layout tables should always be in meaningful sequence by F47. So even if someone decide to implement G57 on sequence level and not for layouttables, it will still fail by F47.
